user_data/strategies/Strategy002.py

```diff
@@ -11,18 +11,23 @@
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 import numpy # noqa
 from datetime import datetime
+import threading
 
-from wao.util import execute, back_test
+from wao.util import Create_Watcher
+from wao._429_file_util import create_429_directory
 from wao.config import Config
-from user_data.strategies.notifier import send_start_deliminator_message
+from wao.notifier import send_start_deliminator_message
 from wao.strategy_controller import StrategyController
 
 
 class Strategy002(IStrategy):
     controller = StrategyController()
+    create_watcher = Create_Watcher()
 
     if Config.IS_BACKTEST:
-        send_start_deliminator_message('Freq Strategy002 ',Config.BACKTEST_COIN ,Config.BACKTEST_MONTH_LIST[Config.BACKTEST_DATA_CLEANER_MONTH_INDEX], Config.BACKTEST_DATA_CLEANER_YEAR, Config.BACKTEST_DUP, Config.BACKTEST_MAX_COUNT_DUP)
+        send_start_deliminator_message('Freq Strategy002 ', Config.BACKTEST_COIN, Config.BACKTEST_MONTH_LIST[Config.BACKTEST_DATA_CLEANER_MONTH_INDEX], Config.BACKTEST_DATA_CLEANER_YEAR, Config.BACKTEST_DUP, Config.BACKTEST_MAX_COUNT_DUP)
+        create_429_directory()
```

Comment:
Not sure how the 429 solution works in execution mode. It seems to be called from backtest mode only

Reply:
I can put it outside the if that way it won't be bound by the condition

Reply:
what is the reason to bound them by the condition

Reply:
I did that based on your suggestion you wanted this message to be sent only if it is on backtest